csaf-import/src/main/kotlin/io/github/csaf/sbom/retrieval/RetrievedProvider.kt

```diff
@@ -99,20 +128,24 @@ class RetrievedProvider(val json: Provider, val role: Role) {
             loader: CsafLoader = lazyLoader
         ): Result<RetrievedProvider> {
             val ctx = ValidationContext()
-            // Closure for providing HttpResponse to ValidationContext.
             // TODO: Only the last error will be available in result. We should do some logging.
             // First, we need to check if a .well-known URL exists.
             val wellKnownPath = "https://$domain/.well-known/csaf/provider-metadata.json"
```

Comment:
Can you please document the contents of https://github.com/csaf-sbom/kotlin-csaf/issues/48#issuecomment-2399612394 here

Reply:
Done.